articles/multifactor-authentication/overview.md

```diff
@@ -0,0 +1,110 @@
+
+End users will be prompted to download whichever app you have enabled in the Dashboard when trying to sign up or log in to your application. Once they indicate that they have successfully downloaded the app, a barcode will appear on screen. They will have a short amount of time in which to scan the code with the designated app. Once this is done, they should see a confirmation screen and also a recovery code to take note of.
+
+Once this is all set up, when the user attempts to authenticate as normal, their device will receive a push notification via the app, and once they approve the request, they will be logged in.
+
+Auth0 supports the use of Auth0 Guardian ([Google Play](https://play.google.com/store/apps/details?id=com.auth0.guardian) / [App Store](https://itunes.apple.com/us/app/auth0-guardian/id1093447833?mt=8)) and Duo ([Google Play](https://play.google.com/store/apps/details?id=com.duosecurity.duomobile) / [App Store](https://itunes.apple.com/us/app/duo-mobile/id422663827?mt=8)) for push notifications.
+
+::: note
+Enabling Duo will result in end users not being given any other choices for MFA factors, even if other factors are enabled and set up.
+:::
+
+#### SMS
+
+SMS as an MFA factor means that the end user is sent a code via SMS when attempting to authenticate with your application. They will have to enter this code to complete the transaction, implying that in addition to knowing their login information, they also have posession of the device set up for receiving MFA texts.
+
+Your users must have a device capable of using SMS to use this option. If your users are unable to always receive SMS messages (such as when traveling), they will be unable to sign up with SMS and will be unable to log in without SMS or their recovery code.
+
+After signing up, and entering a country code and phone number, the user will receive a six digit code to their device. They need to enter this code into the box, and then they will also get a recovery code.
+
+In order to set up SMS, you will need to [configure an SMS provider such as Twilio](/multifactor-authentication/twilio-configuration). You can also [customize your SMS notification templates](/multifactor-authentication/sms-templates).
+
+## Customizing multi-factor authentication
+
+### Customizing via Rules
+
+If you need to customize the multi-factor experience you are offering to your users, you may do so via [custom rules configurations for multi-factor authentication](/multifactor-authentication/custom). This might be needed, for example, if you wish to defined conditions (such as changes in geographic location or logins from unrecognized devices) which would trigger additional authentication challenges.
```

Comment:
Our rules for doing this based on geo location changes were removed. I think we should mention the common cases here that would be triggering MFA for specific applications, or specific users based on metadata, or based on IP.